From 9e81e61548c8bc77765400554c3dce8f7d95e6a2 Mon Sep 17 00:00:00 2001
From: Sandor Yu <Sandor.yu@nxp.com>
Date: Thu, 25 Jan 2018 16:21:52 +0800
Subject: [PATCH 3256/5242] MLK-17461-1: clk: define hdmi pixel select clock

commit  83be80344339e9762181fa9e0d17a2a51b090852 from
https://source.codeaurora.org/external/imx/linux-imx.git

Define hdmi pixel select clocks.
Define av_pll_bypass clock.

Signed-off-by: Sandor Yu <Sandor.yu@nxp.com>
Reviewed-by: Robby Cai <robby.cai@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 include/dt-bindings/clock/imx8qm-clock.h |    8 ++++++--
 1 file changed, 6 insertions(+), 2 deletions(-)

diff --git a/include/dt-bindings/clock/imx8qm-clock.h b/include/dt-bindings/clock/imx8qm-clock.h
index e4720ae..fd7b1b7 100644
--- a/include/dt-bindings/clock/imx8qm-clock.h
+++ b/include/dt-bindings/clock/imx8qm-clock.h
@@ -407,7 +407,6 @@
 #define IMX8QM_DRC1_DIV						372
 #define IMX8QM_DRC1_CLK						373
 
-
 /* HDMI */
 #define IMX8QM_HDMI_AV_PLL_DIV				374
 #define IMX8QM_HDMI_AV_PLL_CLK				375
@@ -835,6 +834,11 @@
 #define IMX8QM_CM41_I2C_CLK				778
 #define IMX8QM_CM41_I2C_IPG_CLK				779
 
-#define IMX8QM_CLK_END					780
+#define IMX8QM_HDMI_PXL_SEL					780
+#define IMX8QM_HDMI_PXL_LINK_SEL			781
+#define IMX8QM_HDMI_PXL_MUX_SEL 			782
+#define IMX8QM_HDMI_AV_PLL_BYPASS_CLK       783
+
+#define IMX8QM_CLK_END					784
 
 #endif /* __DT_BINDINGS_CLOCK_IMX8QM_H */
-- 
1.7.9.5

